File: Basics/Conditional-Statements/Exercises/godzila_vs_kong.py
film_budget = float(input())
statists_count = int(input())
statist_clothes_price = float(input())

movie_decor = film_budget * 0.10
total_clothes_price = statists_count * statist_clothes_price

if statists_count > 150:
    total_clothes_price -= total_clothes_price * 0.10

end_price = total_clothes_price + movie_decor

diff = abs(film_budget - end_price)
if end_price > film_budget:
    print(f"Not enough money!\n"
    f"Wingard needs {diff:.2f} leva more.")
else:
    print(f"Action!\n"
          f"Wingard starts filming with {diff:.2f} leva left.")

File: JavaScript/Basics/0.2-Conditional-Statements/Exercise/godzila-vs-kong.js
function isBudgetEnough(input) {
    let budget = Number(input[0]);
    let statistsCount = Number(input[1]);
    let clothesPrice = Number(input[2]);
    let decorPrice = budget * 0.10;
    clothesPrice *= statistsCount;
    if (statistsCount > 150) {
        clothesPrice -= clothesPrice * 0.10;
    }

    diff = Math.abs(budget - (clothesPrice + decorPrice));
    if (clothesPrice + decorPrice > budget) {
        console.log(`Not enough money!\nWingard needs ${diff.toFixed(2)} leva more.`);
    } else {
        console.log(`Action!\nWingard starts filming with ${diff.toFixed(2)} leva left.`);
    }
}

isBudgetEnough(["20000", "120", "55.5"]);
isBudgetEnough(["15437.62", "186", "57.99"]);
isBudgetEnough(["9587.88", "222", "55.68"]);
